packages/ui/lit-grid/src/dx-grid.ts

```diff
@@ -564,12 +569,14 @@ export class DxGrid extends LitElement {
   override render() {
     const visibleCols = this.visColMax - this.visColMin;
     const visibleRows = this.visRowMax - this.visRowMin;
-    const offsetInline = gap + this.binInlineMin - this.posInline - this.overscanInline;
-    const offsetBlock = gap + this.binBlockMin - this.posBlock - this.overscanBlock;
+    const offsetInline = this.binInlineMin - this.posInline - this.overscanInline;
+    const offsetBlock = this.binBlockMin - this.posBlock - this.overscanBlock;
 
     return html`<div
       role="none"
```

Comment:
not role=grid?

Reply:
I gave `role=grid` to the viewport since focus moves to that if a focused cell is scrolled away… will do another review the a11y of this once all the features are in-place.